LORIS_indepth.html

```diff
@@ -216,63 +94,14 @@ <h4 align="center"><font color="f27c30">
                         </font></h4>
 			<h3 align="center"><a href="https://demo.loris.ca" target="_blank"><span class="fragment">TRY ME!</span></a></h3>
```

Comment:
See my comments on the MNI Ecosystem slide deck for similar/identical slides 
e.g. https://github.com/samirdas/samirdas.github.io/pull/9#discussion_r339785322

Reply:
done